domains/tusharsoni.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Tushar Verma",
+  "repo": "https://github.com/TusharSoni014/is-a.dev",
+  "owner": {
+    "username": "TusharSoni014",
+    "email":"tusharsoni014@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://tusharsoni.com"
```

CNAME doesnt allow trailing slashes (/)
```suggestion
    "CNAME": "tusharsoni.com"
```